gaen-processes.md

```diff
@@ -0,0 +1,40 @@
+# Exposire Notification Processes
+
+This document describes the processes, related to GAEN, used in the app to perform distributed contact registration, letting an infected user notify their digital contacts, and notifying app users of exposure.
+
+## Distributed contact registration
+
+The figure below illustrates how the distributed contact registation works between two different users of an app using the GAEN-framework.
+
+Once actived through consent from the user, the apps will generate daily keys called Temporary Exposure Keys (TEKs). From these keys, a Rotating Proximity Identifier (RPI) is generated every 5-10 minutes. The RPI is then broadcasted through Bluetooth Low Energy to all other devices listening for the same service ID. The broadcasted RPIs are detected by other devices through frequent scanning, and is recorded on the device, along with timestamp and signal strength. This information can later be used to detect exposure from any TEKs linked to infection shared with the device.
```

Comment:
According to markdown best practices, this paragraph `Once actived through consent from the user,...` should be split.
One sentence per one line.